### PR TITLE
Feature/sergeyro/css modules migration 11

### PR DESCRIPTION
**Pull request by SergeyRoyt**
- [ ] 1. Menu
- [ ] 2. MenuDivider
- [ ] 3. MenuGridItem
- [ ] 4. MenuItem
- [ ] 5. MenuItemButton
- [ ] 6. MenuTitle
- [ ] 7. MenuButton

### CODE REVIEW

**Comment by SergeyRoyt** [comment]
Only `Dropdown` snapshots are failing - due to css modules specifics in `react-select`

File: src/components/Menu/MenuGridItem/MenuGridItem.jsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
-import React, { useRef, forwardRef, useCallback } from "react";
-import PropTypes from "prop-types";
 import cx from "classnames";
+import React, { forwardRef, useCallback, useRef } from "react";
+import PropTypes from "prop-types";
 import useMergeRefs from "../../../hooks/useMergeRefs";
-import "./MenuGridItem.scss";
```
Comment (3dyonic) [inline src/components/Menu/MenuGridItem/MenuGridItem.jsx:5]: no css modules file?

File: src/components/Menu/MenuGridItem/MenuGridItem.jsx [hunk 2]
```diff
@@ -73,6 +74,7 @@ const MenuGridItem = forwardRef(
         ref={mergedRef}
         className={cx("menu-grid-item--wrapper", className)}
```
Comment (3dyonic) [inline src/components/Menu/MenuGridItem/MenuGridItem.jsx:75]: no css module class?

**Review by 3dyonic — COMMENTED** [review]
please review my comments

File: src/components/Menu/MenuGridItem/MenuGridItem.jsx (hunk 1 above)
Comment (SergeyRoyt) [reply src/components/Menu/MenuGridItem/MenuGridItem.jsx:5]: Stylesheet contained only this:
`@import "../../../styles/themes";`
So, I've removed it, but left the original classnames to keep existing overrides

File: src/components/Menu/MenuGridItem/MenuGridItem.jsx (hunk 2 above)
Comment (SergeyRoyt) [reply src/components/Menu/MenuGridItem/MenuGridItem.jsx:75]: Stylesheet contained only this:
`@import "../../../styles/themes";`
So, I've removed it, but left the original classnames to keep existing overrides

**Comment by SergeyRoyt** [comment]
Merged into [Vibe2.0](https://github.com/mondaycom/monday-ui-react-core/pull/885)

**Pull request closed without merging**